01-getting-started/getting-started.Rmd

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
Do you have an idea of what the contents of this notebook will look like? Also will this `01-getting-started` directory have a `README.md` file as does the other example analysis directories?

Reply:
Ah the answer to my question is in alexslemonade/refinebio-examples/issues/109 👍